go/libkb/loaduser.go

```diff
@@ -304,7 +304,9 @@ func LoadUserFromServer(g *GlobalContext, uid keybase1.UID, body *jsonw.Wrapper)
 		if err != nil {
 			return nil, err
 		}
-		body = res.Body.AtKey("them")
+		if body, err = res.Body.AtKey("them").ToArray(); err != nil {
```

Comment:
This seems weird to me.  Doesn't rpc_exim.go already translate SCNotFound to NotFoundError?  

I don't think you'd ever get to this line.